index.js

```diff
@@ -479,10 +476,10 @@ function createMergeRequest(options) {
                             open(url);
                           }
                         } else if (mergeRequestResponse.message) {
-                          console.error(colors.red("Couldn't create merge request"));
+                          console.error(colors.red('Couldn\'t create merge request'));
                           console.log(colors.red(mergeRequestResponse.message.join()));
                         } else if (mergeRequestResponse instanceof Array) {
-                          console.error(colors.red("Couldn't create merge request"));
+                          console.error(colors.red('Couldn\'t create merge request'));
```

Comment:
One of the difference which I had covered was error handling. gitlab 3 has better error handling. We need catch handler here instead of looking for message. 
Check https://github.com/vishwanatharondekar/gitlab-cli/compare/master...gitlab-11#diff-168726dbe96b3ce427e7fedce31bb0bcR482 

I like your MR better than mine sine its cleaner for options change and does a bit of eslint cleanup. Just get this change added.